io.github.prateekmedia.appimagepool.yaml

```
@@ -0,0 +1,45 @@
+app-id: io.github.prateekmedia.appimagepool
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
```

Update runtime to 21.08
```suggestion
runtime-version: "21.08"
```